docs/usermanual.md

```diff
@@ -240,6 +240,71 @@ attribute IDs `h` and `t`, then all measures (humidity and temperature) are repo
     $ mosquitto_pub -t /ABCDEF/id_sen1/attrs -m 'h|70|t|15' -h <mosquitto_broker> -p <mosquitto_port> -u <user> -P <password>
 ```
 
+##### Configuration retrieval
+
+The protocol offers a mechanism for the devices to retrieve its configuration (or any other value it needs from those
+stored in the Context Broker). Two topics are created in order to support this feature: a topic for configuration
+commands and a topic to receive configuration information.
```

Comment:
"Parallel" JSON documentation mentions also "This mechanism can be enabled or disabled using a configuration flag, configRetrieval". Is not the same for IOTA-UL?

Reply:
Actually, in the IOTA-JSON code I found that configRetrieval flag affects only the 'subscription' configuration retrieval mechanism. So far I have had no need to introduce it. However, in another PR I am implementing also this second mechanism and the flags is necessary for switch between Bidirectional and Configuration Retrieval mode, that are mutually exclusive.

Reply:
Ok. Thus, it is ok as it is now in this PR and in the new PR the .md will be modified to talk about the flag.

Is my understanding correct?

Reply:
Yes, you are correct.
I will introduce the second mechanism in a new PR because of some issues with the related tests I'm writing for it (actually I am not javascript mother-tongue) and it is taking more time for me to find the bug.
So I have decided to just provide the basic mechanism that sounds to me more suitable for resource-constrained devices. 
The new feature will have of course the documentation updated.

Reply:
OK, great.

NTC